nim/tests/test_routes.py

```diff
@@ -0,0 +1,110 @@
+import json
+from nim.app import app
+import pytest
```

Comment:
According to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports), this should read:
```
import json

import pytest

from nim.app import app
```

Reply:
Done